### PR TITLE
SOC-1867 Make sure we use the user IDs sent into the task

### PR DESCRIPTION
**Pull request by garthwebb**
Previously it was querying for followers again.

https://wikia-inc.atlassian.net/browse/SOC-1867

### CODE REVIEW

File: extensions/wikia/Follow/FollowEmailTask.class.php [hunk 1]
```diff
@@ -4,25 +4,35 @@
 
 class FollowEmailTask extends BaseTask {
 
-	public function emailFollowNotifications( $initiatingUser, $aWatchers, $iUserId, $iNamespace, $sMessage, $sAction ) {
```
Comment (garthwebb) [inline extensions/wikia/Follow/FollowEmailTask.class.php:7]: Removed useless (and in the case of $sValue below, wrong) Hungarian Notation.

File: includes/EmailNotification.php [hunk 2]
```diff
@@ -103,15 +105,19 @@ public function notifyOnPageChange() {
 		}
 
 		// Build a list of users to notify
-		$watchers = [];
```
Comment (garthwebb) [inline includes/EmailNotification.php:106]: This default now happens via the method signature.

**Comment by owend** [comment]
Cool, LGTM

**Comment by jsutterfield** [comment]
:+1:

**Pull request merged**